var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -89,6 +89,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         "hpx_async_dispatch": [False, "Whether HPX supports asynchronous dispath"],
         "tuning": [False, "Create bindings for tuning tools"],
         "tests": [False, "Build for tests"],
+        "compile_as_cmake_language": [False, "Use CMake language support for CUDA/HIP"],
```

Comment:
Option name is quite long, should we try to make it shorter?

Reply:
Sure, we just have to pull it out of the list or make the list smarter. E.g. by adding an optional 3rd element or just directly make it a dict.

Reply:
Shortened it.

Reply:
As per the comment by @cedricchevalier19, and this is outdated, I mark this conversation as resolved.